bloom-housing/bloom
# fix: remove export button from duplicate pages, fix table widths

# Pull Request Template

## Issue Overview

This PR addresses #3002 #3003 #3004

- [x] This change addresses the issues in full

## How Can This Be Tested/Reviewed?

Ensure the duplicates table columns take up the full width of their container, that the old flags tab is gone (+ the old flags pages), and that the export button is removed from the duplicates tabs.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
